lib/widgets/gee_project_editor/gee_project_editor.dart

```diff
@@ -45,99 +48,239 @@ class _GeeProjectEditorState extends StateWithLifecycle<GeeProjectEditor> {
+                widget.width - 100,
+                50,
+                () async {
+                  isNew
+                      ? await _projectEditorStore.postProject(context)
+                      : await _projectEditorStore.updateProject(context);
+                  setState(() {});
                 },
-                initialValue: isNew ? "" : widget.projectRes?.name,
-                decoration: const InputDecoration(
-                  border: OutlineInputBorder(),
-                  labelText: "Project name",
-                  hintText: "Enter project name",
-                  //fill text if isnt new
+                "Save",
+              ),
+            ],
+          ),
+        ),
+      ],
+    );
+  }
+
+  Widget _loadedStatuses() {
+    return ListView.builder(
+      itemCount: _projectEditorStore.statuses.length,
+      itemBuilder: (context, index) {
+        return Row(
+          children: [
+            const SizedBox(width: 20),
+            Container(
+              width: 250,
+              height: 50,
+              decoration: BoxDecoration(
+                color: GeeColors.white,
+                border: Border(
+                  bottom: BorderSide(color: GeeColors.gray1, width: 1),
                 ),
               ),
-            ),
-            const SizedBox(height: 20),
-            SizedBox(
-              // decribtion
-              width: widget.width - 100,
-              height: isNew ? widget.heigth - 390 : widget.heigth - 300,
-              child: TextFormField(
-                keyboardType: TextInputType.multiline,
-                minLines: ((widget.heigth - 300) / 20).round(),
-                maxLines: null,
-                onChanged: (newString) {
-                  _projectEditorStore.description = newString;
-                },
-                initialValue: isNew ? "" : widget.projectRes?.description,
-                decoration: const InputDecoration(
-                  border: OutlineInputBorder(),
-                  labelText: "Project decribtion",
-                  alignLabelWithHint: true,
-                  hintText: "Enter project describtion",
+              child: Center(
+                child: Text(
+                  "${_projectEditorStore.statuses[index].name} (${_projectEditorStore.statuses[index].code.substring(_projectEditorStore.code!.length)})",
+                  style: GeeTextStyles.paragraph2,
                 ),
               ),
             ),
-            const SizedBox(height: 50),
-            //save button
-            geeUniversalButton(
-              widget.width - 100,
-              50,
-              () async {
-                isNew
-                    ? await _projectEditorStore.postProject(context)
-                    : await _projectEditorStore.updateProject(context);
-                setState(() {});
-              },
-              "Save",
-            ),
-          ]),
-        ),
-      ],
+            // const SizedBox(width: 20),
```

Comment:
Jeżeli zakomentowany kod nie jest potrzebny, to bym go usunął

Reply:
Zakomentowana ikonka, do usuwania, w oczekiwaniu na backend